src/frontends/tensorflow_lite/src/sparsity_info.cpp

```diff
@@ -0,0 +1,163 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "sparsity_info.hpp"
+
+#include "schema_generated.h"
+
+bool ov::frontend::tensorflow_lite::SparsityInfo::is_copyable() const {
+    return false;
+}
+
+// Unpacks sparse data. Supports only case when sparse tensor has only one sparse dimension [DENSE, ..., DENSE, SPARSE]
+template <typename T, typename U>
+static void read_sparse_data(uint8_t* dest,
+                             uint8_t* dest_end,
+                             const size_t row_size,
+                             const ::flatbuffers::Vector<T>* indices,
+                             const ::flatbuffers::Vector<U>* segments) {
+    U* data = reinterpret_cast<U*>(dest - row_size);  // row size will be increased at first step
+    T last_idx = ~static_cast<T>(0);
+    for (auto idx = indices->begin(); idx != indices->end(); ++idx) {
+        if (*idx <= last_idx) {
+            data = reinterpret_cast<U*>(reinterpret_cast<uint8_t*>(data) + row_size);
+            FRONT_END_GENERAL_CHECK(reinterpret_cast<uint8_t*>(data) < dest_end, "Dense data is out of bounds");
```

Comment:
I think we need to replace `reinterpret_cast`, security checks will alarm us about it.

Reply:
I'll think about it, thanks!

Reply:
Replaced by static_cast only